Libraries/LibWeb/Credentials/NavigatorCredentialsContainer.idl

```diff
@@ -0,0 +1,7 @@
+#import <Credentials/CredentialsContainer.idl>
+
+// FIXME: This is a partial interface not an interface mixin
```

Comment:
oh, I didn't notice that this was specced as a partial interface. I don't think we support those in the IDL generator... I think for previous partial interfaces we've cheated and just added the method/attributes to the main idl file for the interface with a reference to the spec that is extending it :sweat_smile: 

That would certainly make the managment of the 'container' class a lot easier, if it didn't exist. Sorry for the wrong direction on this one.